examples/petsc/build_ex.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
This stray file should be removed.

Reply:
Whoops, committed that by accident.  Removed.